mem/table.go

```diff
@@ -72,3 +72,7 @@ func (t *Table) Insert(row sql.Row) error {
 	t.data = append(t.data, row.Copy())
 	return nil
 }
+
+func (t Table) String() string {
+	return fmt.Sprintf("Table(%s)", t.name)
```

Comment:
Maybe should we print the schema here too?

Reply:
Sure